src/entity.ts

```diff
@@ -42,6 +45,9 @@ export function decrypt<T extends ObjectLiteral>(entity: any): any {
     let { propertyName, mode, target } = columnMetadata;
     let options: ExtendedColumnOptions = columnMetadata.options;
     let encrypt = options.encrypt;
+    if (encrypt?.encryptionPredicate && !encrypt?.encryptionPredicate(entity)) {
```

Comment:
can you merge this with `let encrypt = options.encrypt` above or with the condition below?